scripts/aws-auto-backup.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const AWS = require('aws-sdk');
+const { execSync } = require('child_process');
+const path = require('path');
+const argv = require('minimist')(process.argv.slice(2));
+const { lstatSync, readdirSync } = require('fs');
+const { join } = require('path');
+
+
+if (!argv.config) {
+    argv.config = '.origintrail_noderc';
+}
+
+if (!argv.configDir) {
```

Comment:
Please use one kind of naming convention for argv, you have configDir and backup_rirectory